### ISSUE TITLE
Implement recently used list in the icon styling component [v0.5]

### ISSUE THREAD

**Issue by javierarce** [CartoDB/cartodb#11021]
Implement recently used list in the icon styling component  [BLOCKED BY #10746]

![screen shot 2016-12-12 at 16 48 06](https://cloud.githubusercontent.com/assets/4933/21105512/c6e405d0-c08a-11e6-842a-2bcf23ff7c74.png)

> Clicking on the Img link, the panel will change to allow icon selection. Icons on the RECENTLY USED list will be ordered depending on the last time an icon was used, or by default if none has been used yet. In the future, would be great to have global metrics by usage, so we can offer the most popular icons instead of showing them in a discrete order - out of scope for this version 

This feature will require some kind of persistence to know which icons are selected by the user.

See workflow and design in [this issue](https://github.com/CartoDB/design/issues/303). 

Final design is [here](https://github.com/CartoDB/design/issues/303#issuecomment-262497147)

**Comment by alrocar** [comment]
Stale issue. Closing 👋
